### DISCUSSION TITLE
Remove heating/cooling duty

### DISCUSSION THREAD

**Discussion by zoroch** [thorrak/fermentrack#613] [Ideas]
Hello,
Is it possible to have an option to hide the heating and cooling duty to make the graphic more readable ?
![image](https://user-images.githubusercontent.com/80644355/111775856-92e4f700-88b1-11eb-8001-e7d54951d694.png)
I like that you can hide the different temperatures, would be great to do the same to the cooling duty

**Comment by cdnninja** [comment]
I thought you can be click the item on the right side?

**Reply by vanosg** [reply]
Resurrecting a dead thread only because it annoys me sometimes too :) Not for those two items, I believe it's a legacy "feature" from the original BrewPi.

**Reply by thorrak** [reply]
All good - I'm going to tackle this in Fermentrack 2
